README.md

```diff
@@ -3,13 +3,13 @@ Project to run the whole ggtracker stack in vagrant
 
 ### Running
  * Make sure you have vagrant+virtualbox installed on your computer
- * Run `vagrant up` to create the virtual box
+ * Run `git submodule update --init` to pull submodules
+ * Run `vagrant up` to create the virtual box. This will download all required packages and go through the installation process for ggtracker and ESDB (~15 minutes; go make yourself a sandwich)
  * Run `vagrant ssh` to ssh into the vagrant box
- * Find code in /vagrant
- * Run the installation and updating steps of the components (ESDB and ggtracker) ([will be automated soon](https://github.com/gravelweb/ggtrackerstack/commit/e3bdbb30d9384d37e8b96692af81cefd5f8a87d2))
-   * ESDB: https://github.com/dsjoerg/esdb#installation-and-setup
-   * ggtracker: https://github.com/dsjoerg/ggtracker#basic-installation-and-updating
-     * Note: You need to run ggtracker webapp like this `ESDB_HOST=172.28.128.3:9292 foreman start` (change ip accordingly)
+ * Find code in /vagrant `cd /vagrant`
+ * Start the application (requires 2 ssh sessions, or run them as background jobs)
+   * ESDB: `(cd esdb && ESDB_HOST=172.28.128.3:9292 foreman start)` (change IP accordingly)
```

Comment:
Isn't this the other way around - ESDB_HOST is a parameter for ggtracker's `foreman start`?